demos/sslecho/main.c

```diff
@@ -335,10 +335,8 @@ int main(int argc, char **argv)
     if (server_skt != -1)
         close(server_skt);
 
-    if ( txbuf && (txcap>0) )
-      {
-	free(txbuf);
-      }
+    if (txbuf != NULL && txcap >0)
```

Comment:
One final nit: Please add space after `>`.

Reply:
forced commit with this included.